pytest_tests/testsuites/acl/test_eacl.py

```diff
@@ -679,3 +680,29 @@ def test_extended_actions_system(self, wallets, eacl_container_with_objects):
                     endpoint=endpoint,
                     wallet_config=storage_wallet.config_path,
                 )
+
+
+    @pytest.mark.trusted_party_proved
+    @allure.title("Not owner and not trusted party can NOT set eacl")
+    def test_only_owner_can_set_eacl(
+        self,
+        wallets,
```

Comment:
It is a good thing to use type annotations as much as possible, even when it is some fixtures. We can specify here a type that is returned from fixtures.

Reply:
> It is a good thing to use type annotations as much as possible, even when it is some fixtures. We can specify here a type that is returned from fixtures.

Fixed